cmd/otel-agent/subcommands/run/command.go

```diff
@@ -163,6 +162,12 @@ func runOTelAgentCommand(ctx context.Context, params *subcommands.GlobalParams,
 		fx.Invoke(func(_ collectordef.Component, _ defaultforwarder.Forwarder, _ optional.Option[logsagentpipeline.Component]) {
 		}),
 
+		configsyncimpl.OptionalModuleWithParams(),
+		fx.Provide(func() configsyncimpl.Params {
+			return configsyncimpl.NewParams(params.SyncTO, params.SyncDelay, true)
+		}),
+		fx.Invoke(func(_ optional.Option[configsync.Component]) {}),
```

Comment:
What is this doing?

Reply:
As the component is instantiated [here](https://github.com/DataDog/datadog-agent/pull/28319/files#diff-61728f1ab33613f3a3106cf2cdf99ea65b22e1a09566afbac256c79164cab69fR183) this is not needed.

Reply:
Yup, forgot cleaning up.